lib/Distribution/Sle/16Latest.pm

```diff
@@ -13,4 +13,16 @@ use parent Distribution::Sle::AgamaDevel;
 use strict;
 use warnings FATAL => 'all';
 
+use Yam::Agama::Pom::GrubMenuBasePage;
```

Comment:
we could solve this code duplicity by creating:
`get_grub_menu_base` on Leap with a condition for aarch there.
then `get_grub_menu_installed_system` still need to be created for each product by is just one line.

Because current code would miss to apply this in `lib/Distribution/Sle/AgamaDevel.pm`

Reply:
Thanks, updated code and VRs.